modules/bot.py

```diff
@@ -22,6 +24,72 @@ def init():
     allowoffline = [] #users with !allowoffline
     waiting_reactions = dict() #{message_id: function}
 
+
+class UnpickedPool():
```

Comment:
This new data structure makes it really easy to work with players:
* It assigns each player a 'position'
* it provides easy to use functions that condense the usage to one or two calls when the unpicked pool is referenced